addOns/ascanrules/src/main/java/org/zaproxy/zap/extension/ascanrules/PathTraversalScanRule.java

```diff
@@ -659,6 +647,40 @@ public int getWascId() {
         return 33;
     }
 
+    private AlertBuilder createUnmatchedAlert(String param, String attack, String evidence) {
+        return newAlert()
+                .setConfidence(Alert.CONFIDENCE_LOW)
+                .setParam(param)
+                .setAttack(attack)
+                .setEvidence(evidence)
+                .setOtherInfo(Constant.messages.getString("ascanrules.pathtraversal.other", 5))
+                .setAlertRef(getId() + "-" + 5);
+    }
+
+    private AlertBuilder createMatchedAlert(
+            String param, String attack, String evidence, int check) {
+        return newAlert()
+                .setConfidence(Alert.CONFIDENCE_MEDIUM)
+                .setParam(param)
+                .setAttack(attack)
+                .setEvidence(evidence)
+                .setOtherInfo(Constant.messages.getString("ascanrules.pathtraversal.other", check))
+                .setAlertRef(getId() + "-" + check);
+    }
+
+    @Override
+    public List<Alert> getExampleAlerts() {
+        List<Alert> list = new ArrayList<>(5);
+        for (int checkNum = 1; checkNum < 5; checkNum++) {
+            list.add(
+                    createMatchedAlert(
+                                    "query", "file:///c:/Windows/system.ini", "file.ext", checkNum)
```

Comment:
These values are not accurate for all alerts.

Reply:
Okay I'll have to go through things in more detail.

Reply:
I’ll try to finish this up tomorrow.